pkg/table/table.go

```diff
@@ -37,6 +37,49 @@ func Write(headers []string, data [][]interface{}) {
 	table.Render()
 }
 
+func AppendRow(row []interface{}, colWidth []int) {
+	table := tablewriter.NewWriter(os.Stdout)
+
+	table.SetRowLine(false)
```

Comment:
wrap repeating table configuration in a function

Reply:
okay.

Reply:
done. please check